app/views/users/deletion_info.html.erb

```diff
@@ -26,28 +26,36 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-<%= toolbar title: "#{l('account.deletion_info.heading', :name => h(@user.name))}" %>
-<%= labelled_tabular_form_for :user, :url => user_path(@user), :html => { :method => :delete, :class => 'confirm_required' } do |form| %>
-
+<%= labelled_tabular_form_for :user, url: user_path(@user), html: { method: :delete, class: 'confirm_required form danger-zone' } do |form| %>
   <div class='wiki'>
-    <p>
-      <%= l("account.deletion_info.login_consequences.#{User.current == @user ? 'self' : 'other'}") %>
-    </p>
+    <section class="form--section">
+      <h3 class="form--section-title">
+        <%= l('account.deletion_info.heading', name: "<em>#{h(@user.name)}</em>").html_safe %>
+      </h3>
 
-    <p>
-      <%= l("account.deletion_info.data_consequences.#{User.current == @user ? 'self' : 'other'}") %>
-    </p>
+      <p>
+        <%= l("account.deletion_info.login_consequences.#{User.current == @user ? 'self' : 'other'}") %>
+      </p>
 
-    <p>
-      <%= l("account.deletion_info.info.#{User.current == @user ? 'self' : 'other'}") %>
-    </p>
+      <p>
+        <%= l("account.deletion_info.data_consequences.#{User.current == @user ? 'self' : 'other'}") %>
+      </p>
+      <p class="danger-zone--warning">
+        <span class="icon icon-attention2"></span>
+        <span><%= l("account.deletion_info.info.#{User.current == @user ? 'self' : 'other'}") %></span>
+      </p>
+      <p>
+        <%= l("account.deletion_info.login_verification.#{User.current == @user ? 'self' : 'other'}", name: "<em>#{h(@user.name)}</em>").html_safe %>
+      </p>
+      <div class="danger-zone--verification">
+        <input type="text" name="login_verification"/>
+        <%= form.button '', class: 'button -highlight' do
+          concat content_tag :i, '', class: 'button--icon icon-delete'
+          concat content_tag :span, l(:button_delete), class: 'button--text'
+          end %>
```

Comment:
This block looks suspiciously hand-woven. Did you have a look at `submit_tag` or `styled_button_tag`? Because as far as I know, those can be customized too. You may look [here](https://github.com/opf/openproject/pull/3451/files#diff-b970740d79353d77d89b9c8f6fbdaa90L43) for inspiration.